libs/parser/parser.c

```diff
@@ -365,14 +366,12 @@ int parser_execute(parser_t *p, const char *path)
 		}
 	}
 
-	res = 0;
-
 	if (feof(file) == 0) {
-		fprintf(stderr, "%s: error on file parsing\n", FUNCTION_EXECUTE);
+		fprintf(stderr, "%s: error on parsing %s\n", FUNCTION_EXECUTE, path);
```

Comment:
Please remove defines with function names, and use `__FUNCTION__`
```suggestion
		fprintf(stderr, "%s: error on parsing %s\n", __FUNCTION__, path);
```